doc/DeveloperGuide.md

```diff
@@ -94,6 +94,10 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. Should never [crashes] (#crashes) during usage
+6. Should have specialized interfaces for [different devices] (#different devices)
```

Comment:
NFR should be specific or tangible, so you better describe more about the `specialized interface`.